pkg/oci/containerd.go

```diff
@@ -304,7 +305,8 @@ func (c *Containerd) GetManifest(ctx context.Context, dgst digest.Digest) ([]byt
 	return b, mt, nil
 }
 
-func (c *Containerd) CopyLayer(ctx context.Context, dgst digest.Digest, dst io.Writer) error {
+func (c *Containerd) CopyLayer(ctx context.Context, dgst digest.Digest, dst io.Writer, bsize int) error {
```

Comment:
This _is_ a public function. Is there any backward compatibility guarantees expected here @phillebaba?

Reply:
I am a bit hesitant to expose this configuration pattern. If we are going to switch to using copy with buffer we should initially set a default value, unless there is a specific reason to expose it.

As getting the layer size is possible we could have a threshold for the buffer size depending on the amount of data being copied.

Reply:
there is default value: 32768 (32KB), which is the default buffer size for io.copy. I use this value as default value to minimize the impact to current user. User don't need change helm chart value if they are ok with default value. 

> As getting the layer size is possible we could have a threshold for the buffer size depending on the amount of data being copied.

Good idea, this can be thought as a feature request in the future.